infrastructure/main.tf

```diff
@@ -446,13 +448,15 @@ resource "aws_cloudfront_distribution" "api_gateway" {
   default_cache_behavior {
     allowed_methods  = ["GET", "HEAD"]
     cached_methods   = ["GET", "HEAD"]
-    target_origin_id = "get-stale-object"
-    cache_policy_id  = aws_cloudfront_cache_policy.get_stale_object.id
-
+    target_origin_id = aws_apigatewayv2_api.lambda_stale_object.id
+    cache_policy_id  = var.test_stale_object ? aws_cloudfront_cache_policy.get_stale_object.id : "4135ea2d-6df8-44a3-9df3-4b5a84be39ad"
```

Comment:
This part contain magic string, I think it's better to have a variable for 4135ea2d-6df8-44a3-9df3-4b5a84be39ad, you could try locals, prefer data_source if it can be retrieve from data source

Reply:
Thanks for your comment! Hard-coding cache policy has been replaced by data source.